### PR TITLE
Errors in openshift pipeline

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1540223

### CODE REVIEW

**Comment by bparees** [comment]
lgtm

**Comment by mburke5678** [comment]
@dongboyan77 PTAL
In the end, I removed the "experimental wording" from the Jenkins page in the 3.7 documentation. 
The Jenkins page in the 3.7 docs is currently:
https://docs.openshift.com/container-platform/3.7/using_images/other_images/jenkins.html#client-plugin-in
The OpenShift Client Plugin is installed in OpenShift by default and the `oc new-app jenkins-ephemeral` should work in 3.7 out-of-the-box.
Please let me know if these changes are correct.

**Comment by dongboyan77** [comment]
LGTM, thanks

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL. 

The OpenShift Client Plugin feature went Supported in 3.7, but we had is described as "experimental." I took the wording from 3.9 and applied it to the 3.7 docs.

File: using_images/other_images/jenkins.adoc [hunk 1]
```diff
@@ -628,28 +628,26 @@ plug-in for creating CI/CD flows for {product-title} in Jenkins.
...
 
+
 [[client-plugin-in]]
 == {product-title} Client Plug-in
 
-The experiences gained working with users of the OpenShift Pipeline plug-in,
-coupled with the rapid evolution of both Jenkins and OpenShift, have provided
-valuable insight into how to integrate {product-title} from Jenkins jobs.
+The OpenShift Container Platform Client Plug-in aims to provide a readable, concise, comprehensive, 
+and fluent Jenkins Pipeline syntax for rich interactions with OpenShift Container Platform.
```
Comment (kalexand-rh) [inline using_images/other_images/jenkins.adoc]: maybe s/OpenShift Container Platform/{product title}

File: using_images/other_images/jenkins.adoc [hunk 2]
```diff
@@ -628,28 +628,26 @@ plug-in for creating CI/CD flows for {product-title} in Jenkins.
...
+
 [[client-plugin-in]]
 == {product-title} Client Plug-in
 
-The experiences gained working with users of the OpenShift Pipeline plug-in,
-coupled with the rapid evolution of both Jenkins and OpenShift, have provided
-valuable insight into how to integrate {product-title} from Jenkins jobs.
+The OpenShift Container Platform Client Plug-in aims to provide a readable, concise, comprehensive, 
+and fluent Jenkins Pipeline syntax for rich interactions with OpenShift Container Platform. 
+The plug-in leverages the `oc` binary, which must be available on the nodes executing the script.
```
Comment (kalexand-rh) [inline using_images/other_images/jenkins.adoc]: s/leverages/uses

**Review by kalexand-rh — COMMENTED** [review]
Two suggestions, otherwise LGTM

**Pull request merged**